contracts/src/token/erc20/mod.rs

```diff
@@ -286,6 +290,13 @@ impl IErc20 for Erc20 {
     }
 }
 
+impl IErc165 for Erc20 {
```

Comment:
I am not sure if we want to have it implemented always.

Reply:
Basically `Erc20` compare to `Erc721` doesn't have `Erc165` support. I think original issue is [here](https://github.com/OpenZeppelin/openzeppelin-contracts/issues/1073#issuecomment-478674541) was closed due to "complexity". Some other implementations of Erc20 standard (not open zeppelin) have it.
Also seeking around github by actual Erc20 interface id `0x36372b07`, I could found find projects that rely on the Erc20 with interface support.
In fact at stylus contracts we are not forcing users to expose `supportsInterface(bytes4)`. But they have an option to do so.
@ggonzalez94 what do you think?

Reply:
https://github.com/OpenZeppelin/rust-contracts-stylus/pull/281#discussion_r1772048963
^^